src/store.js

```diff
@@ -0,0 +1,16 @@
+import { createStore, compose, applyMiddleware } from 'redux';
+import { promiseMiddleware } from 'promise-middleware-redux';
+import thunk from 'redux-thunk';
+import reducer from './reducers';
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+export default createStore(
+  reducer,
+  composeEnhancers(
+    applyMiddleware(
+      thunk,
+      promiseMiddleware
```

Comment:
promiseMiddleware shouldn't be implemented on this branch - it should just be on the next one